.github/actions/setup/action.yml

```diff
@@ -6,24 +6,22 @@ runs:
   steps:
     - name: Setup Node
       uses: actions/setup-node@v4
-      with:
-        node-version: 'lts/iron'
+      with: { node-version-file: .nvmrc }
 
     - name: Install PNPM
-      uses: pnpm/action-setup@v2
+      uses: pnpm/action-setup@v4
       with:
         run_install: false
 
-    - name: Get PNPM store path
-      id: pnpm-cache
+    - name: Get pnpm store directory
       shell: bash
       run: |
-        echo "pnpm_cache_dir=$(pnpm store path)" >> $GITHUB_OUTPUT
+        echo "PNPM_STORE_PATH=$(pnpm store path)" >> $GITHUB_ENV
```

Comment:
What's the reason for putting this in the env instead of output?

Reply:
Copied what pnpm docs suggest latest. No other reason.